OnlineStoreTCA/Root/RootDomain.swift

```diff
@@ -38,7 +38,21 @@ struct RootDomain: ReducerProtocol {
         uuid: { UUID() }
     )
     
-    var body: some ReducerProtocol<State, Action> {
+    static let demo = Self(
```

Comment:
We do have live and test instances in the project already. I don't think we need these new declarations.